src/components/Team/TeamDialog.tsx

```diff
@@ -95,7 +95,7 @@ export function TeamDialog(props: IProps) {
   const { anySaving, toolsChanged, startSave, clearRequested } =
     useContext(UnsavedContext).state;
   const { getBible, getBibleOwner, getOrgBible } = useBible();
-  const { canAddPublishing } = useCanPublish();
+  const { userCanPublish } = useUserCanPublish();
```

Comment:
the only question I have here is that the norm is that everything that starts with "use" is a hook. Perhaps we should name it just "canPublish"?

Reply:
or maybe "canUserPublish" is better?